samples/drivers/adc/adc_dt/sample.yaml

```diff
@@ -32,6 +32,7 @@ tests:
       - longan_nano
```

Comment:
you can combine your testscope changes and the overlay into a single commit

Reply:
ok, I will combine them, thanks for your suggestions.

Reply:
yes, please combine all the test changes to one commit since they are so simple